README.md

```diff
@@ -1 +1,56 @@
-# security-kibana-plugin
+[![CI](https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/workflows/CI/badge.svg?branch=master)](https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/actions)
+
+# Open Distro for Elasticsearch Security Kibana Plugin
+
+This plugin for Kibana adds a configuration management UI for the Open Distro for Elasticsearch Security features, as well as authentication, session management and multi-tenancy support to your secured cluster.
+
+## Basic features
+
+* Kibana authentication for Open Distro for Elasticsearch
+* Kibana session management
+* Open Distro for Elasticsearch Security configuration UI
+* Multi-tenancy support for Kibana
```

Comment:
Can we also add 
* Audit log support for Kibana

Reply:
sure, will add it